resolwe_bio/processes/alignment/star.yml

```diff
@@ -507,7 +507,7 @@
         {% if advanced.genomeSAsparseD %}--genomeSAsparseD {{ advanced.genomeSAsparseD }}{% endif %} \
         --genomeDir ./star_index/ \
         --genomeFastaFiles genome.fasta \
-        --sjdbGTFfile ${ANNOT} \
+        --sjdbGTFfile "${ANNOT}" \
```

Comment:
This will fail for annotations that are not in GTF format. See how `ANNOT` is set a few lines above.

Reply:
Nice catch. I have prepared a fix + updated the default .gff3 test file used by `prepare_annotation_gff` to account for whitespace also in this scenario.